### ISSUE TITLE
Failing readthedocs

### ISSUE THREAD

**Issue by arnfinn** [PCMSolver/pcmsolver#163]
Failed with this commit: 809e4a9a

Reason:
Line 83 in `doc/conf.py` throws an error since `prere` is *NoneType*.

Error:
```
  File "conf.py", line 83, in <module>
    tweak = '+'.join([prere, sha])
TypeError: sequence item 0: expected str instance, NoneType found
```
Dirty fix:
```
-tweak = '+'.join([prere, sha])
+try:
+    tweak = '+'.join([prere, sha])
+except:
+    tweak = sha
```

**Comment by robertodr** [comment]
Yep. I am on it. I will also have to add "check that docs build" to the
release management documentation.

**Comment by arnfinn** [comment]
Do we run `sphinx-build . _build` with Travis to see if a PR/commit break readthedocs? If not, maybe we could add it. Maybe also run [htmlproofer](https://github.com/gjtorikian/html-proofer) afterwards.

**Comment by arnfinn** [comment]
Fixed here: e882df9275
